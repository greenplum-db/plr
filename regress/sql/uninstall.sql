\! gppkg -q --all | sed -n '/^plr-[0-9]/s|.*|plr|p'
\! echo '############# SEPARATOR LINE ###########'
-- start_ignore
\! gppkg -r plr
-- end_ignore
\! gppkg -q --all | sed -n '/^plr-[0-9]/s|.*|plr|p'
