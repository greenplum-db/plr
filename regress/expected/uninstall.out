\! gppkg -q --all | sed -n '/^plr-[0-9]/s|.*|plr|p'
plr
\! echo '############# SEPARATOR LINE ###########'
############# SEPARATOR LINE ###########
-- start_ignore
\! gppkg -r plr
-- end_ignore
\! gppkg -q --all | sed -n '/^plr-[0-9]/s|.*|plr|p'
